tools/api-markdown-documenter/src/api-item-transforms/Utilities.ts

```diff
@@ -106,8 +108,15 @@ function resolveSymbolicLink(
 		apiModel.resolveDeclarationReference(codeDestination, contextApiItem);
 
 	if (resolvedReference.resolvedApiItem === undefined) {
+		const linkSource =
```

Comment:
Previously, links from the package-level `@packageDocumentation` comment printed the source as an empty string. The new logic handles this better, and also carves out a label for the unexpected case where the associated package can't be found for an API member.